src/tiles/clickable-tile.component.ts

```diff
@@ -0,0 +1,70 @@
+import {
+	Component,
+	Input
+} from "@angular/core";
+
+/**
+ * Build application's clickable tiles using this component.
+ *
+ * ## Basic usage
+ *
+ * ```html
+ * <ibm-clickable-tile>
+ * 		tile content
+ * </ibm-clickable-tile>
+ * ```
+ *
+ * @export
+ * @class ClickableTile
+ * @implements {OnInit}
+ */
+@Component({
+	selector: "ibm-clickable-tile",
+	template: `
+	<a
+		[ngClass]="{
+			'bx--link': true,
+			'bx--tile': true,
+			'bx--tile--clickable': true,
+			'bx--tile--is-clicked': clicked
+		}"
+		tabindex="0"
+		(click)="onClick($event)"
+		(keydown)="onKeyDown($event)"
+		[href]="href"
+		[attr.aria-disabled]="disabled">
+		<ng-content></ng-content>
+	</a>`
+})
+export class ClickableTile {
+	static CARRIAGE_RETURN = "Enter";
```

Comment:
These are no longer magic values (like `13` was) so are we getting any benefit from assigning it to static variables?

Reply:
Seconded, the strings alone are pretty self-explanatory I feel